_posts/_2021-08-31-building-opensearch-1.1-distributions.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "Building OpenSearch 1.1 Distributions"
+authors: 
+  - dblock
+date: 2021-08-31
+categories: 
+  - technical-post
+twittercard:
+  description: "This post details how we produce reliable and repeatable distributions of OpenSearch."
+---
+The OpenDistro for Elasticsearch (ODFE) was essentially a package containing Elasticsearch OSS and a dozen open-source plugins. Its release process began by picking up a stable version of Elasticsearch OSS, incrementing all the plugin version numbers, and going through a development cycle. Plugins went from `-SNAPSHOT` to `-beta`, then `-rc`, then finally to a release.
+
+This worked well until the [OpenSearch fork](https://aws.amazon.com/blogs/opensource/introducing-opensearch/). It was decided that OpenSearch would be a product with two distributions at launch: a full distribution that includes a dozen plugins, and a minimal distribution without any plugins. This made sense because most customers either wanted a well tested distribution with everything, or are technically savvy enough to assemble their favorite flavor of all the things. 
```

Comment:
- "well tested" -> "well-tested"
- "customers" -> "users"
- I don't care for "technically savvy enough" because it implies that those who don't do this aren't technically savvy.